src/util/FidoUtil.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint complexity:[2, 9] */
+
+define(function () {
+  var fn = {};
+
+  fn.getU2fEnrollErrorMessageKeyByCode = function (errorCode) {
```

Comment:
I think it will be better if we change these 2 functions for U2F error messages to return the actual message instead of the key, since everywhere these functions are used is just to feed the key to an `Okta.loc` fn
```
fn.getU2fEnrollErrorMessage = function (errorCode) {
  ...
  return Okta.loc(key, 'login');
}